optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/test/java/org/optaplanner/quarkus/OptaPlannerProcessorOverridePropertiesAtRuntimeTest.java

```diff
@@ -40,12 +41,14 @@
 
 import io.quarkus.test.QuarkusUnitTest;
 
-public class OptaPlannerProcessorOverrideTerminationAtRuntimeTest {
+public class OptaPlannerProcessorOverridePropertiesAtRuntimeTest {
 
     @RegisterExtension
     static final QuarkusUnitTest config = new QuarkusUnitTest()
-            // We want to check if this is overriden by our runtime alternative
+            // We want to check if these are overridden by our runtime alternative
             .overrideConfigKey("quarkus.optaplanner.solver.termination.best-score-limit", "0")
+            .overrideConfigKey("quarkus.optaplanner.solver.move-thread-count", "4")
```

Comment:
Where do we test if this actually ends up as 4?

Reply:
We don't; we test if this actually ends up at 3 (i.e. the runtime config of 3 overrides the build time config of 4)